### ISSUE TITLE
Proposal: change underlying name of SARIF from Static ... to Systematic ...

### ISSUE THREAD

**Issue by Motional-Charles-Wilson** [oasis-tcs/sarif-spec#603]
For the next major release of SARIF the underlying name should be changed from **Static Analysis Report Interchange Format** to **Systematic Analysis Report Interchange Format**. This would better reflect the scope and applicability of the standard. This change would be in support of the additional non-static analysis use cases being recommended for support in the next major revision.

**Comment by sthagen** [comment]
I think we discussed this over a year ago in quite some length weighing charter change needs against good naming effects. I remember we decided to postpone until publication of the errata (which is now passed).

I am in favor of discussing this issue but I am -1 o the "Systematic" as this sounds to me too sales like similar to "Clean" or "Optimal" ... but we will find for sure the best path forward for the SARIF ... I think it is simple as long as the first word starts with "S" or has an "s" at a prominent place (as the "x" in "Extensible" Markup Language --> XML)

**Comment by davidmalcolm** [comment]
Some possible candidates that begin with an "s":

- system
- systemic
- systematic
- SARIF (as a recursive acronym)
- software (but too limited in scope)

**Comment by davidmalcolm** [comment]
Did we decide on "System" here?

**Comment by sthagen** [comment]
@davidmalcolm Yes, System it is. We documented that decision in https://github.com/oasis-tcs/sarif-spec/blob/main/meeting_minutes/240606_SARIF_TC_88.md#52-review-of-decisions-reached-secretary-stefan
